src/vic.h

```diff
@@ -13,10 +13,12 @@
 #define VIC_BUFFER_SIZE 128
 #define VIC_FUNCS_COUNT 64
 #define VIC_FUNC_NAME_LEN 6
+#define VIC_VARS_COUNT 64
+#define VIC_VAR_NAME_LEN 6
 
 #define VIC_ERR_NO 0
-#define VIC_ERR_FUNC_INSUFFICIENT_SPACE 1
-#define VIC_ERR_FUNC_WRONG_NAME 2
+#define VIC_ERR_INSUFFICIENT_SPACE 1
+#define VIC_ERR_WRONG_NAME 2
```

Comment:
@vargac if we are changing names of constants I suggest we change this one to `VIC_ERR_INVALID_NAME`. The reason is as follows: when this error is returned it is not brcause the name provided would be wrong, but that it is not valid (too long...). Therefore I believe the name of this constant should suggest that (it is not valid) by containing the term `INVALID`.

Does that make  sense?